website/cli.html

```diff
@@ -0,0 +1,348 @@
+<!DOCTYPE html>
```

Comment:
Minor: Run `cd website && yarn fmt` to prettify this HTML code. (Don't bother running `yarn fmt` from `generate-cli-docs`; our other code generation scripts don't run the formatter either.)